app/routes.jsx

```diff
@@ -1,13 +1,27 @@
 import React from "react";
 import { Route, IndexRoute } from "react-router";
+import {trackAndOpenLinkInCurrentTab } from "./helpers/handleGA";
 // containers
 import RootComponent from "./components/root";
 import OldHome from "./oldhome";
 import Home from "./home";
 import NoPage from "./components/404";
 import FAQContainer from "./faq";
 
+export const WHITE_PAPER_ADDRESS = "https://pluto-asset.s3-accelerate.amazonaws.com/PLUTO+whitepaper_ver_0.3.pdf";
```

Comment:
I don't think that it is not a good solution to delivering whitepaper to both writer and reader. We have to make autonomous manageable way. For example, add lambda function to search latest pdf at S3 & give way to upload whitepaper to a writer.

Reply:
I've thought about this. However, there should be the point where a developer intervene. For example, the PDF maker should know about 'How to upload to S3' and 'namespace rule of S3' too.
I think the current way and your way both can solve the problem neither.